float32 traj_inertia 
---
uint32 active_errors
uint8 procedure_result
